assets/css/src/comments.css

```diff
@@ -159,7 +159,7 @@
 .comment-reply-link:focus,
 .comment-form .form-submit input:hover,
 .comment-form .form-submit input:focus {
-	border-color: #000;
+	border-color: var(--global-font-color);
```

Comment:
For all the borders, I think it would be better to define a new variable `--global-border-color` and use that. It could use the same color like `--global-font-color` by default, but it'd be great so separate that.

Reply:
Thanks for the suggestion (applied).